### DISCUSSION TITLE
Resource Manager Hiding Permissions Tab from Non-Administrators

### DISCUSSION THREAD

**Discussion by cduluCNB** [dnnsoftware/Dnn.Platform#6137] [Q&A]
Hi all,

I'm just wondering if there's a way to disable the permissions tab for non-administrator users? Ideally this non-admin user should only be able to add/modify/delete folders and files, but currently they're also able to edit the permissions on these files and folders.

Been looking for an answer to this, but can't seem to find anything on this. 

![2024-09-26 09_07_53-Assets](https://github.com/user-attachments/assets/650165d8-784b-4e93-9052-8efe02c8c43b)

**Comment by valadas** [comment]
Well, I think that is to support "group" mode. Now if you are not an admin or in a place where you should be able to edit permissions, please use https://github.com/dnnsoftware/Dnn.Platform/security/advisories/new
